src/js/aframe/aframeHelper.js

```diff
@@ -0,0 +1,42 @@
+const aframeHelper = {
+  async init() {
+    await this.getData();
+  },
+
+  async getData() {
+    const data = await storeChild.getAll();
+    const { nodes, links } = this.transformData(data);
```

Comment:
could you just return this here (instead of creating these intermediate constant variables)?